.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,225 @@
+name: Publish Docker Image
+
+on: [push, pull_request]
```

Comment:
Is `pull_request` needed?

Reply:
No, as try to split into multiple jobs (e.g amd64 and arm64 ) (feedback from daipom), temporary enabled for a while.

Reply:
I see.